### PR TITLE
Table header scrolls with the doc manifest

### PR DESCRIPTION
**Pull request by Sjones352**
Connects #574 
### AC
1. As a user I would like to be able to see the column labels when I scroll through the list of files eX is retrieving. Currently the table headings scroll with the doc list.

header

### CODE REVIEW

File: app/views/help/_faqs_with_vva.html.erb [hunk 1]
```diff
@@ -141,7 +140,8 @@
   <h2 id="encounter-problems">17. What should I do if I encounter problems?</h2>
 
   <p>
-    If you encounter any problems while using eFolder Express, you should ask your supervisor for assistance. If you and your supervisor are unable to resolve the issue, please reach out to the Caseflow Product Support Team by calling 1-844-876-5548 or emailing caseflow@va.gov.
+    If you encounter any problems while using eFolder Express, you should ask your supervisor for assistance. If you and your supervisor are unable to resolve the issue, please reach out to the Caseflow Product Support Team by calling 1-844-876-5548 or emailing <a href="mailto: caseflow@va.gov">caseflow@va.gov.</a>
+  </p>
```
Comment (mmlumba) [inline app/views/help/_faqs_with_vva.html.erb]: You might have added this extra P tag on accident

Comment (Sjones352) [inline app/views/help/_faqs_with_vva.html.erb]: No it was not an accident. I wanted to change the caseflow.gov email to a link. see screenshot below
screen shot 2017-08-10 at 12 58 14 pm

Comment (mmlumba) [inline app/views/help/_faqs_with_vva.html.erb]: You don't need another `<p>` tag within a paragraph unless you're creating a new paragraph.

File: app/views/help/_faqs_with_vva.html.erb [hunk 2]
```diff
@@ -153,5 +152,5 @@
   <h2 id="need-help">19. What if I still need help?</h2>
 
   <p>
-    If you still encounter issues or have questions that are not answered here, reach out to the Caseflow Product Support Team by calling 1-844-876-5548 or emailing caseflow@va.gov.
+    If you still encounter issues or have questions that are not answered here, reach out to the Caseflow Product Support Team by calling 1-844-876-5548 or emailing<a href="mailto: caseflow@va.gov">caseflow@va.gov.</a>
```
Comment (mmlumba) [inline app/views/help/_faqs_with_vva.html.erb]: There also needs to be a space between emailing and the hyper link

**Pull request merged**